tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/TokenMapping.sol

```diff
@@ -388,6 +396,8 @@ contract TokenMapping is TokenQueryLogic {
 		tokenAddresses["uni"][Web3Networks.Bsc] = Uni.getTokenBscAddress();
 		tokenAddresses["uni"][Web3Networks.Ethereum] = Uni
 			.getTokenEthereumAddress();
+		tokenAddresses["uni"][Web3Networks.Solana] = Uni
+			.getTokenSolanaAddress();
```

Comment:
I feel we don't need the mapping variable tokenNames, because we can use tokenNetworks to check whether request tokenName exist or not instead.
i.e. tokenNetworks[tokenName].length > 0